egs/wsj/s5/steps/cleanup/append_eval_to_ctm.py

```diff
@@ -0,0 +1,396 @@
+#!/usr/bin/env python
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0
+
+from __future__ import print_function
+import sys, operator, argparse
+
+# Append to the CTM the Levenshtein alignment of 'hypothesis' and 'reference':
+# (i.e. the output of 'align-text' post-processed by 'wer_per_utt_details.pl')
+
+# The tags in the processed Levenshtein alignment are:
+#  'C' = correct
+#  'S' = substitution
+#  'D' = deletion
+#  'I' = insertion
+#  'EC' = silence and neighboring words are both correct
```

Comment:
I don't understand the naming convention here, why E?
You should explain at this point what the precedence is among the types of error.  For instance, del sil sub, what tag does that put on the silence?
What does it mean for a neighboring word to be a deletion, given that a deletion  does not appear in a transcript?  You might need to develop more precise language for this (or explain further below).   Does you add extra lines to the CTM for deletions?   Does 'silence' mean any phone that is in phones/silence.txt, or just the optional silence?  
Is it really necessary to add this information at this point, given that much of it is easy to work out from the CTM?
Be clear about what you mean by an OOV-- presumably a reference word that does not appear in words.txt.  Try to be explicit what you mean about  'words' here (e.g. in 'neighboring words'), given that there are two word sequences involved.